template-quality-checklist.md

```diff
@@ -0,0 +1,102 @@
+# Template quality checklist
+
+This quality checklist will eventually become a pull request template that every contributor needs to add to pull requests against the `templates` repository.
+
+
+# Pull request summary
+
+{Describe the purpose of your pull request.}
+
+## Template contributor checklist
+
+IMPORTANT: The next three sections in this checklist should be filled out by the template contributor at the time they submit this request.
+
+NOTE: Pull requests can only be merged when all boxes are checked.
+
+### Which issue does this pull request fix or reference?
+
+This pull request:
+
+- Resolves: {Choose this option if this PR will close the issue and link to issue}
+- Relates to: {Choose this option if this PR only references an issue for context but does not fully resolve it}
+
+### Procedural requirements
+
+- [ ] The template contributor participated in a template working group to create this template.
+- [ ] This template was workshopped with the community.

```

Comment:
I'd avoid using the term workshopped and say "This template was reviewed by at least three community members." Though as noted in one of my other comments, I think two members is sufficient for a new template and one member sufficient for a revision to an existing template.

Reply:
Noted, but I prefer 3. See my previous comment.

Reply:
I've adopted this suggestion.